cli/aiconfig-editor/src/shared/prompt_schemas/OpenAIChatModelParserPromptSchema.ts

```diff
@@ -66,7 +66,6 @@ export const OpenAIChatModelParserPromptSchema: PromptSchema = {
         type: "integer",
         maximum: 4096,
         minimum: 16,
-        default: 4096,
```

Comment:
Removing this for now, seems like a default value will be persisted if it exists